das_client/lib/app/pages/journey/train_journey/widgets/header/time_container.dart

```diff
@@ -1,40 +1,52 @@
-import 'package:sbb_design_system_mobile/sbb_design_system_mobile.dart';
+import 'package:das_client/app/bloc/train_journey_cubit.dart';
 import 'package:flutter/material.dart';
+import 'package:das_client/model/journey/journey.dart';
+import 'package:sbb_design_system_mobile/sbb_design_system_mobile.dart';
 
 class TimeContainer extends StatelessWidget {
   const TimeContainer({super.key});
 
   @override
   Widget build(BuildContext context) {
-    return SBBGroup(
-      margin: const EdgeInsetsDirectional.fromSTEB(
-        sbbDefaultSpacing * 0.5,
-        0,
-        sbbDefaultSpacing * 0.5,
-        sbbDefaultSpacing,
-      ),
-      padding: const EdgeInsets.all(sbbDefaultSpacing),
-      useShadow: false,
-      child: SizedBox(
-        width: 124.0,
-        height: 112.0,
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-          crossAxisAlignment: CrossAxisAlignment.center,
-          children: [
-            Text(
-              '05:43:00',
-              style: SBBTextStyles.largeBold.copyWith(fontSize: 24.0),
+    final bloc = context.trainJourneyCubit;
+
+    return StreamBuilder<Journey?>(
+        stream: bloc.journeyStream,
+        builder: (context, snapshot) {
+          if (!snapshot.hasData || snapshot.data == null) {
+            return Center(
+              child: SBBLoadingIndicator(),
+            );
+          }
+          final Journey journey = snapshot.data!;
+
+          return SBBGroup(
+            margin: const EdgeInsetsDirectional.fromSTEB(
+              sbbDefaultSpacing * 0.5,
+              0,
+              sbbDefaultSpacing * 0.5,
+              sbbDefaultSpacing,
             ),
-            _divider(),
-            Text(
-              '+00:01:30',
-              style: SBBTextStyles.largeLight.copyWith(fontSize: 24.0),
+            padding: const EdgeInsets.all(sbbDefaultSpacing),
+            useShadow: false,
+            child: SizedBox(
+              width: 124.0,
+              height: 112.0,
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  Text(
+                    '05:43:00',
```

Comment:
Das ist nicht in deiner Story drin, aber du könntest hier noch die aktuelle Systemzeit anzeigen.